### PR TITLE
Add one AHCI disk case

### PR DESCRIPTION
**Pull request by chunfuwen**
No error happened when do vm.save and snapshot with ahci disk

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virtual_disks.multidisks.coldplug.single_disk_test.disk_option_bootdisk.disk_virtio_snapshot.disk_snapshot_external.ahci_disk_test

File: libvirt/tests/cfg/virtual_disks/virtual_disks_multidisks.cfg [hunk 1]
```diff
@@ -497,6 +497,16 @@
                                         - normal_test:
                                             snapshot_error = "no"
                                             snapshot_option = "--disk-only"
+                                        - ahci_disk_test:
```
Comment (dzhengfy) [inline libvirt/tests/cfg/virtual_disks/virtual_disks_multidisks.cfg:500]: As AHCI is Intel only , please help add 'only x86_64'.

Comment (dzhengfy) [reply libvirt/tests/cfg/virtual_disks/virtual_disks_multidisks.cfg:500]: Others LGTM.

Comment (chunfuwen) [reply libvirt/tests/cfg/virtual_disks/virtual_disks_multidisks.cfg:500]: updated

**Pull request merged**